extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/DevConsoleCORSFilter.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.logging.Logger;
+
+import io.quarkus.vertx.http.runtime.cors.CORSConfig;
+import io.quarkus.vertx.http.runtime.cors.CORSFilter;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.ext.web.RoutingContext;
+
+public class DevConsoleCORSFilter implements Handler<RoutingContext> {
```

Comment:
That does not look like complete CORS support, mostly an origin check.
I would feel better if clearly stated it, or if we implement CORS completely.

Reply:
@cescoffier As I said in the previous comment, this filter offers a complete CORS support plus the specific localhost origin check. By complete support, in scope of the DevConsole work, `Origin` is completely checked, with or without preflight requests

Reply:
Yeah! I missed the delegation.